src/components/HeroBanner/_heroBanner.scss

```diff
@@ -2,22 +2,21 @@
 
 .#{$px}-hero-banner {
   align-items: center;
-  background: var(--background);
+  background: linear-gradient(rgba(0, 0, 0, 60%), rgba(0, 0, 0, 60%)), var(--background);
```

Comment:
Are we sure we want to hard code the linear gradient? I know it sucks to write these but if we wanted to just provide a linear gradient as a background for whatever reason we will not be able to overwrite this. I do not think we have a current use case, just wondering if we want to keep the flexibility.

Reply:
I'm assuming they'll want this consistency, but I think we adjust later if needed.